CHANGELOG.md

```diff
@@ -104,9 +106,9 @@ We'd love to get feedback.
 
 ## 2.2.0-alpha
 
-Add nullable reference types support (Sentry, Sentry.Protocol) (#509) 
-fix: Use ASP.NET Core endpoint FQDN (#485) 
-feat: Add integration to TaskScheduler.UnobservedTaskException (#481) 
+Add nullable reference types support (Sentry, Sentry.Protocol) (#509)
```

Comment:
I think you editor doesn't like the empty spaces, it changed all files you saved

Reply:
oops, will change that back.